playbooks/templates/k8s/namespace/04_portal.yml.j2

```diff
@@ -102,6 +102,9 @@ data:
     REDIS_PORT=26379
     REDIS_QUEUE=default
     REDIS_SENTINELS=tcp://application-redis:26379
+    {%- for i in range(0, wai_redis_slave_count) -%}
+      ,tcp://application-redis:26379
+    {%- endfor %}
```

Comment:
Il servizio di kubernetes maschera gli endpoint dei sentinel e di redis il cui numero totale è in realtà #master + #slave quindi potrebbe essere più corretto aggiungere la configurazione che hai fatto anche per i master

Reply:
Dal chart helm avevo dedotto che la configurazione con sentinel è sempre 1 master + `n` slave, per questo non ho previsto una variabile per il numero dei master.

Reply:
Corretta l'osservazione del #master [Master-Slave-With-Sentinel](https://github.com/helm/charts/tree/master/stable/redis#master-slave-with-sentinel) ma il range potrebbe essere range(0, wai_redis_slave_count, 1) in modo da creare 3 entry nella variabile ([jinja](http://jinja.palletsprojects.com/en/2.10.x/templates/#range))